### DISCUSSION TITLE
2 Suggestions!

### DISCUSSION THREAD

**Discussion by MIfoodie** [SchmarrnDevs/Lighty#15] [Ideas and Suggestions]
1. Hotkey (F6 maybe) to quickly turn on and off Lightly without using the menu, it will use the last selected setting, or if no setting was previously set it uses carpet mode.
2. A mode that allows the rendering of Lighty carpets on the four sides of a block.

**Comment by andi-makes** [comment]
Thank you for your suggestions!

1. You are lucky, the code for this is already implemented, I just need to release it. The default keybind will be F8, but you can change that in the keybinds menu of Minecraft :D
2. Would this only affect Blocks that already have Lighty carpets on them in the normal carpet mode? Like, if you are standing in front of a flat wall, only the upmost blocks would get the side overlay? Or should it be rendered on every block and the side shows the light level of the block it is facing?
